src/AiPlayer.js

```diff
@@ -56,12 +56,60 @@ var ticTacToe = ticTacToe || {};
     }
   };
 
-  app.AiPlayer.prototype.calculateMove = function (board, side) {
+  app.AiPlayer.prototype.calculateMove = function (board, me) {
+    var enemy = this.getEnemy(me);
+    // stub to make the Ai always return a move...
+    var square = this.isSquareAvailable(board);
+    if (square !== false) return square;
+    // end stub. remove when coding commences.
+    this.isSquareAvailable(board);
+    var twoInARow = this.findTwoInARowWithEmptyOption(board, me);
+    if (twoInARow !== false) return twoInARow;
+    return false;
+    throw new Error('could not decide on a move');
+  };
+
+  app.AiPlayer.prototype.isSquareAvailable = function (board) {
     for (var i = 0; i < this.getNumberOfSquares(); i++) {
       if (board[i] === 0) {
-        return i;
+        return board[i];
       }
     }
-  };
+    throw new Error("there are no available squares to play");
+  }
+
+  app.AiPlayer.prototype.findTwoInARowWithEmptyOption = function (board, side) {
```

Comment:
Can you add YUI documentation to all of your methods and properties within the class?

http://yui.github.io/yuidoc/